pkg/deploy/cmd/delete_test.go

```diff
@@ -40,10 +42,7 @@ func TestStop(t *testing.T) {
 		replicationControllerKind      = schema.GroupVersionKind{Kind: "ReplicationController"}
 	)
 
-	pause := func(d *deployapi.DeploymentConfig) *deployapi.DeploymentConfig {
-		d.Spec.Paused = true
-		return d
-	}
+	pauseBytes := []byte(`{"spec":{"paused":true,"replicas":0,"revisionHistoryLimit":0}}`)
```

Comment:
Same for tests, manually changing this is too fragile.

Reply:
i think handcrafting is better choice here since it cuts some latency from the delete request and getting a DC here just to generate exactly the same string everytime does not justify the `CreateTwoWayMergePatch` I guess.

Reply:
I'll stay strong against manual handcrafting these type of patches. They might bite us at some point.